Plugins/Public/autobuy/Main.cpp

```diff
@@ -217,7 +222,29 @@ void LoadSettings()
 				{
 					if (ini.is_value("item"))
 					{
-						mapAutobuyFLHookExtras[CreateID(ini.get_value_string(0))] = CreateID(ini.get_value_string(1));
+						mapAutobuyFLHookCloak[CreateID(ini.get_value_string(0))] = CreateID(ini.get_value_string(1));
```

Comment:
we shouldn't be recording all extra entries as cloaks

Reply:
We should not. But in reality those were used as cloak and cloak disruptor batteries

Reply:
This is what had default autobuy.cfg few months, which makes all those Extra Entities

[extra]
item = dsy_cloak_1, missile01_mark01_ammo
item = dsy_cloak_2, missile01_mark01_ammo
item = dsy_cloak_3, missile01_mark01_ammo
item = dsy_cloak_4, missile01_mark02_ammo
item = dsy_cloak_1_n, missile01_mark04_ammo
item = dsy_cloak_2_n, missile01_mark04_ammo
item = dsy_cloakdisruptor_1, missile01_mark01_ammo
item = dsy_cloakdisruptor_2, missile01_mark01_ammo
item = dsy_cloakdisruptor_3, missile01_mark01_ammo